spark-plugins/docs/LogisticRegressionClassifier-sparkcompute.md

```diff
@@ -0,0 +1,70 @@
+# LogisticRegressionClassifier
+
+
+Description
+-----------
+Loads a Logistic Regression model from a file of a FileSet dataset and uses it to classify records.
```

Comment:
why is it limited to File Set? Can we extend it to get input from other sources too such as streams and tables?